src/DataService/PublicIdentifierInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Lullabot\Mpx\DataService;
+
+/**
+ * Interface definition for all mpx objects with a public identifier.
+ */
+interface PublicIdentifierInterface
+{
+    /**
+     * Returns the public identifier for this mpx object.
+     *
+     * @return string
+     */
+    public function getPid(): string;
+}
```

Comment:
Wouldn't it be appropriate to include setPid() in this interface too?

Reply:
I'd actually removed it over at https://github.com/Lullabot/mpx-php/pull/101/commits/19d5af2f68c33af74fa5bd96e545ab4217f3bbd4

However, I realized that while splitting the interface makes things a little simpler for read-only clients, it makes them more complex for clients that want to modify data. I'll put it back!